dbt/tests/data_tests/generic_tests/expect_column_fraction_with_condition.sql

```diff
@@ -1,20 +1,30 @@
 {% test expect_column_fraction_with_condition(
     model,
     column_name,
-    column_condition,
-    expected_fraction,
-    tol=0.02
+    row_condition,
+    min_fraction,
+    max_fraction
```

Comment:
For most of the other tests where there's some range we've gone with explicitly setting a min/max rather than a central value with a tolerance -- it's just a little more flexible/explicit -- so I made that tweak here.

There's already a convention in `dbt` (or maybe it's `dbt_expectations`) that this kind of selector is called named `row_condition` rather than `column_condition`, so using the latter seemed kind of confusing to me. That being said, the behavior here is slightly different than the existing `row_condition` default meaning -- in the other tests it preemptively limits what rows are sent into the test at all, so maybe we should pick another name entirely.

Reply:
(though I realize the original test you were migrating had this symmetric check)

Reply:
Yeah I saw that `dbt_expectations` has the `row_condition` convention, but also felt like this case didn't exactly mirror the behavior used there. I then must have got confused about intuiting what a `column_condition` would be. `row_condition` seems fine and I think is intuitive enough, but maybe `row_filter` is a better name just to be unambiguous about it being a little different than `dbt_expectations`?